### PR TITLE
Karlburke/files api

### PR DESCRIPTION
**Pull request by kburke**
search-api endoints to work with Elasticsearch index of Dataset file manifests.  Soon to be integrated with files-api revisions.

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -19,10 +19,8 @@
 # Log rotation is handled via logrotate on the host system with a configuration file
 # Do NOT handle log file and rotation via the Python logging to avoid issues with multi-worker processes
 
-logging.basicConfig(format='[%(asctime)s] %(levelname)s in %(module)s:%(lineno)d: %(message)s', level=logging.DEBUG,
```
Comment (yuanzhou) [inline src/app.py:22]: Need to revert the change, DEBUG level is being used and it's helpful for troubleshooting.

File: src/libs/es_writer.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 # All the API logging is forwarded to the uWSGI server and gets written into the log file `uwsgo-entity-api.log`
 # Log rotation is handled via logrotate on the host system with a configuration file
 # Do NOT handle log file and rotation via the Python logging to avoid issues with multi-worker processes
-logging.basicConfig(format='[%(asctime)s] %(levelname)s in %(module)s:%(lineno)d: %(message)s', level=logging.DEBUG, datefmt='%Y-%m-%d %H:%M:%S')
```
Comment (yuanzhou) [inline src/libs/es_writer.py:11]: Same as above, need DEBUG level to print out the detailed error messages.

**Review by yuanzhou — CHANGES_REQUESTED** [review]
@kburke I don't know much about the actual changes/functionalities. Just requested to revert the DEBUG level.

File: src/app.py (hunk 1 above)
Comment (kburke) [reply src/app.py:22]: Logging level reverted to DEBUG in branch file.

File: src/libs/es_writer.py (hunk 2 above)
Comment (kburke) [reply src/libs/es_writer.py]: Logging level reverted to DEBUG in branch file.

**Review by yuanzhou — APPROVED** [review]

**Review by maxsibilla — APPROVED** [review]
